ElasticsearchReportHelper.php

```diff
@@ -386,6 +410,12 @@ public static function download(array $options) {
       ],
       $indicia_templates['button']
     );
+    if (isset($controlOptions)) {
```

Comment:
Rather than inline CSS, these divs should be styled by updating media/scss/common/_indicia.datacomponents.scss and rebuilding the css files.

Reply:
I will change.